### DISCUSSION TITLE
GitHub Accounts Flagged as Spam - Assistance Need

### DISCUSSION THREAD

**Discussion by HarvinConsulting** [community/community#140712] [Other Feature Feedback, Questions, & Ideas] [answered]
Dear GitHub Support Team,

I hope you are well. I am writing to seek assistance regarding an issue I’ve encountered while creating new GitHub accounts. I recently purchased a domain e.g (xxx.com) and created email addresses under this domain, such as:

xxx@xxx.com
xxx@xxx.com
Using these email addresses, I created GitHub accounts. However, both accounts were immediately flagged as spam. I believe this might be due to automated systems detecting the domain or the account creation process as suspicious, though there was no ill-intent or unusual activity from my side.

Details of My Setup:

The domain is newly purchased, and the emails have been properly verified.
SPF, DKIM, and DMARC records have been configured for the domain to ensure authenticity.
The accounts were created manually without the use of any automation or scripting tools.
Despite this, the accounts were marked as spam. I would appreciate your help in understanding why this happened and guidance on how to prevent such issues when creating new accounts in the future.

If needed, I can provide additional details regarding the domain setup or account activities. Please let me know how we can proceed to resolve this.

Thank you for your time and support. I look forward to your assistance in this matter.

Best regards,
Aftab Khan

**Comment by queenofcorgis** [comment, marked as answer]
Hi @HarvinConsulting ! The best place to go for answers to account-specific questions like this one is opening a ticket on our [Support page](https://support.github.com/contact). 

The community will be unable to assist with nor escalate your ticket, but we can assure you it is in the right place. It will be answered in the order it was received based on our team's resources and the amount of tickets ahead of yours. We appreciate your understanding! 

As no other Community Discussions team members will be able to provide additional help with account related questions I am going to close this discussion.

Thank you!

_Please note for future discussions: You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them._
